website/pages/playground/index.html

```diff
@@ -318,57 +313,36 @@ <h1>Prettier <span id="version"></span></h1>
   </header>
 
   <div class="editors-container">
-    <div class="editors">
-      <div class="editor loading input">
-        <textarea id="input-editor"></textarea>
-      </div>
-      <div class="editor loading ast">
-        <textarea id="ast-editor"></textarea>
-      </div>
-      <div class="editor loading doc">
-        <textarea id="doc-editor"></textarea>
-      </div>
-      <div class="editor loading output">
-        <textarea id="output-editor"></textarea>
-      </div>
-      <div class="editor loading output2">
-        <textarea id="output2-editor"></textarea>
-      </div>
-    </div>
-  </div>
-
-  <div class="bottom-bar">
-    <div class="bottom-bar-buttons">
-      <button type="button" class="btn" id="button-clear">
-        Clear
-      </button>
-    </div>
-
-    <details class="options-details">
-      <summary class="options-summary">Options</summary>
+    <details id="options-details" class="options-container">
+      <summary class="options-summary"></summary>
+      <div class="options">
+        <details class="sub-options">
+          <summary>Global</summary>
```

Comment:
It'd be better to open (`<details open>`) `Global` and `JavaScript` by default I think.

Reply:
Yeah that could work.. I'd rather persist the last state though.

(That'd be so easy to in a react component 😅)